utils/cms.js

```diff
@@ -54,6 +56,68 @@ export function getCmsPages(state) {
   return state.records.filter(record => { return record.type === 'page' && record.enabled })
 }
 
+
+export function getCmsNav(state) {
+  return state.records.filter(record => { return record.type === 'nav' && record.enabled })
+}
+
+
+const AccordionNav = ({nav, pages}) => {
+  const [open, setOpen] = useState(false);
+  const aria_id = `basic-nav-section-${nav.key}`;
+  // Create a ref that we add to the element for which we want to detect outside clicks
+  const ref = useRef();
+
+  // Call hook passing in the ref and a function to call on outside click
+  useOnClickOutside(ref, () => setOpen(false));
+
+  const menuOpenStyles = {
+    display: open ? 'inline' : ''
+  }
+  return <li key={nav.key} className="usa-nav__primary-item" ref={ref}>
+    <button className="usa-accordion__button usa-nav__link"
+            aria-expanded={open}
+            aria-controls={aria_id}
+            onClick={() => setOpen(!open)}>
+    <span>{nav.title}</span></button>
+    {open ? <ul id={aria_id} className="usa-nav__submenu">
+      {
+        pages.map((page) => {
+          return <li className="usa-nav__submenu-item">
+            <a className="usa-nav__link" href={`/${page.key}`}><span>{page.title}</span></a>
+          </li>
+        })
+      }
+    </ul>: undefined}
```

Comment:
Would { open && <ul ... } work better for this, so we don't have to set the undefined?